### ISSUE TITLE
Sorting columns

### ISSUE THREAD

**Issue by theevilbit** [redcanaryco/mac-monitor#7]
**At a high level -- can you summarize your request?**
It would be nice to be able to order columns alphabetically when clicking on the column name.

**What is the current alternative solution?**
Export the events and do it in another tool

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
None, just general filtering/sorting

**Anything else?**
NA

**Comment by Brandon7CC** [comment]
I gotcha!

**Comment by Brandon7CC** [comment]
Feature added in v1.9.0!
